internal/commands/manifest_create.go

```diff
@@ -0,0 +1,50 @@
+package commands
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/buildpacks/pack/pkg/client"
+	"github.com/buildpacks/pack/pkg/logging"
+)
+
+// ManifestCreateFlags define flags provided to the ManifestCreate
+type ManifestCreateFlags struct {
+	format            string
+	insecure, publish bool
+}
+
+// ManifestCreate creates an image-index/image-list for a multi-arch image
+func ManifestCreate(logger logging.Logger, pack PackClient) *cobra.Command {
+	var flags ManifestCreateFlags
+
+	cmd := &cobra.Command{
+		Use:   "create <manifest-list> <manifest> [<manifest> ... ] [flags]",
+		Args:  cobra.MatchAll(cobra.MinimumNArgs(2), cobra.OnlyValidArgs),
+		Short: "Create a manifest list or image index.",
+		Example: `pack manifest create cnbs/sample-package:hello-multiarch-universe \
+		cnbs/sample-package:hello-universe \
+		cnbs/sample-package:hello-universe-windows`,
+		Long: `Generate manifest list for a multi-arch image which will be stored locally for manipulating images within index`,
+		RunE: logError(logger, func(cmd *cobra.Command, args []string) error {
+			return pack.CreateManifest(
+				cmd.Context(),
+				client.CreateManifestOptions{
+					IndexRepoName: args[0],
+					RepoNames:     args[1:],
+					Format:        flags.format,
+					Insecure:      flags.insecure,
+					Publish:       flags.publish,
+				},
+			)
+		}),
+	}
+
+	cmdFlags := cmd.Flags()
+
+	cmdFlags.StringVarP(&flags.format, "format", "f", "v2s2", "Format to save image index as ('OCI' or 'V2S2')")
+	cmdFlags.BoolVar(&flags.insecure, "insecure", false, "Allow publishing to insecure registry")
```

Comment:
```suggestion
	cmdFlags.BoolVar(&flags.insecure, "insecure", false, "When pushing the index to a registry, do not use TLS encryption or certificate verification")
```

This is a weird user interface IMO. `insecure` should be something you pass in when pushing the index, not when instantiating it.

Reply:
We wanted to allow some end users to create an index and push it to the registry right away, for example, you already ran a `pack build` for your app image (amd64, arm64) for example, and you just want to do everything in one operation.

`pack manifest create my-index my-app:arm64 my-app:amd64 --publish` 

In this case, you don't need to keep the local index, maybe useful for CI/CD pipelines